concent_api/middleman/middleman_server.py

```diff
@@ -1,10 +1,11 @@
 import asyncio
-from asyncio.base_events import BaseEventLoop
+from asyncio import BaseEventLoop  # type: ignore
```

Comment:
Is there a difference with those two ? If yes and it causes some problems then I am sorry, because I have changed this to avoid the mypy ignore.

Reply:
quite frankly, I was playing with different options and eventually have left one. I will restore your solution.
done;